### PR TITLE
docs: add note to Array.prototype.find() page

### PR DESCRIPTION
**Pull request by tomaszwysocki**
### Description
Add note to Array.prototype.find() page

### Motivation
Added a note to the Array.prototype.find() page suggesting the use of Array.prototype.filter() when looking for all elements that satisfy the provided testing function.

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/array/find/index.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ If no values satisfy the testing function, {{jsxref("undefined")}} is returned.
 - If you need to find if a value **exists** in an array, use {{jsxref("Array/includes", "includes()")}}.
   Again, it checks each element for equality with the value instead of using a testing function.
 - If you need to find if any element satisfies the provided testing function, use {{jsxref("Array/some", "some()")}}.
+- If you need to find if all elements that satisfy the provided testing function, use {{jsxref("Array/filter", "filter()")}}.
```
Comment (dipikabh) [inline files/en-us/web/javascript/reference/global_objects/array/find/index.md:19]: - Note the use of "if"
- There is possibility to add two separate statements, one for `every()` and another for `filter()`:

```suggestion
- If you need to find if all elements satisfy the provided testing function, use {{jsxref("Array/every", "every()")}}.
- If you need to find all elements that satisfy the provided testing function, use {{jsxref("Array/filter", "filter()")}}.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for your contribution, @tomaszwysocki.
Please check the suggested edits.

File: files/en-us/web/javascript/reference/global_objects/array/find/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/array/find/index.md:19]: @dipikabh `filter` is related to this method; `every`'s relationship is too far-fetched. Someone who thinks about `find` won't be looking for `every`. They might be looking for `some`.

Comment (dipikabh) [reply files/en-us/web/javascript/reference/global_objects/array/find/index.md]: Got it! Let's remove `every` in that case. The original addition "if all elements that satisfy" was confusing.

File: files/en-us/web/javascript/reference/global_objects/array/find/index.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ If no values satisfy the testing function, {{jsxref("undefined")}} is returned.
 - If you need to find if a value **exists** in an array, use {{jsxref("Array/includes", "includes()")}}.
   Again, it checks each element for equality with the value instead of using a testing function.
 - If you need to find if any element satisfies the provided testing function, use {{jsxref("Array/some", "some()")}}.
+- If you need to find if all elements satisfy the provided testing function, use {{jsxref("Array/every", "every()")}}.
```
Comment (dipikabh) [inline files/en-us/web/javascript/reference/global_objects/array/find/index.md:19]:
```suggestion

```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Remove `every` as per https://github.com/mdn/content/pull/31981#discussion_r1476736396

**Review by dipikabh — APPROVED** [review]

**Pull request merged**